### PR TITLE
Seal of Approval additions

### PR DESCRIPTION
**Pull request by kbodwin**
PR'ing to add the Seal_of_Approval.md document that keeps a list of approved packages, as well as proposing the first three pilot packages for approval.

(Also updating R Build Ignore and README accordingly.)

### CODE REVIEW

File: Seal_of_Approval.md
Comment (Anirban166) [inline Seal_of_Approval.md:11]: Maybe 'Named capture groups in regular expressions ... '?

Comment (tdhock) [reply Seal_of_Approval.md:11]: this was copied from https://github.com/tdhock/nc

**Comment by tdhock** [comment]
looks good to me, thanks Kelly.
Since there are some significant changes to the README, and I am involved with this PR, I will not merge this.
Can another committer please review and merge? @jangorecki @ben-schwen

File: Seal_of_Approval.md
Comment (jangorecki) [inline Seal_of_Approval.md:21]: This link is wrong, no?

**Comment by jangorecki** [comment]
Looks good, left one minor comment

File: Seal_of_Approval.md
Comment (TysonStanley) [reply Seal_of_Approval.md:21]: I believe this is referring to the six that we list in the governance.

- Time & memory efficiency
- Concise syntax (minimal redundancy in code)
- No external Imports/LinkingTo/Depends dependencies (external meaning those not maintained by the project)
- Few (if any) Suggests/Enhances dependencies
- Stable code base (strong preference for user-friendly back-compatibility with data.table itself and with old versions of R)
- Comprehensive and accessible documentation and run-time signals (errors, warnings)

File: Seal_of_Approval.md
Comment (TysonStanley) [reply Seal_of_Approval.md:17]: It is designed differently than `dtplyr` so I'm fine with how it is currently stated.

**Review by TysonStanley — APPROVED** [review]
LGTM

File: Seal_of_Approval.md
Comment (jangorecki) [reply Seal_of_Approval.md:21]: can we link that section of the file?

Comment (tdhock) [reply Seal_of_Approval.md]: I did

**Comment by tdhock** [comment]
great thanks!

**Review by jangorecki — COMMENTED** [review]
Lgtm

**Pull request merged**
